### ISSUE TITLE
Grammarly Mistakes

### ISSUE THREAD

**Issue by TabNahida** [Bylx666/bylx666#2]
You should check your grammar. It's too terrible.
1. ``nobody knows, however``->``Nobody knows. However, ``
2. ``I am a energy reducer`` -> ``I am an energy reducer`` Actually, I cannot understand what is an energy reducer. Energy saver would be better.
3. ``You cannot preview what I am going to do, so do myself.`` I can get what you want to say, but the word you use is too ``CHINGLISH``. ``Preview`` -> ``predict`` ``myself`` -> ``I``.
4. ``My name is Subkey Charisma, means the secondary Key`` -> ``My name is Subkey Charisma, which means the secondary Key``
5. ``So it is essencial but not important?!`` -> ``So it is essential but not important!``
6. ``With balanced feeling`` -> ``With a balanced feeling`` However, it's strange.
7. ``frames`` -> ``framework``
## CHECK YOUR GRAMMAR PLEASE!!!!!!
It's too difficult for me to understand your sentences. I recommend you to use [grammarly](https://grammarly.com). It's free. And I think that would improve your English. If I'm your English teacher, I would go to hell immediately.

**Comment by Bylx666** [comment]
哈哈中国人就讲中国话呗, 毕竟我一直以为我主页不会有人看所以没咋在意细节语法.

**Comment by TabNahida** [comment]
You're right, but Genshin Impact is a new open-world adventure developed by HoYoverse itself. The game takes place in a fantasy world called Teyvat, where those chosen by the gods are granted the "Vision" , the power of the guiding elements. You will play a mysterious character named "Traveler", in the free travel to meet different personalities, unique abilities of the companions, together with them to defeat the enemy, find lost relatives -- and gradually discover the truth of the "Allogene".

**Comment by kde-yyds** [comment]
> You're right, but Genshin Impact is a new open-world adventure developed by HoYoverse itself. The game takes place in a fantasy world called Teyvat, where those chosen by the gods are granted the "Vision" , the power of the guiding elements. You will play a mysterious character named "Traveler", in the free travel to meet different personalities, unique abilities of the companions, together with them to defeat the enemy, find lost relatives -- and gradually discover the truth of the "Allogene".

玩原神玩的
